### PR TITLE
Feature/add classes

### PR DESCRIPTION
**Pull request by sergio-usma**
**### Project requirements**
A class called Person was to be created with the following features:

- Instance variables: @id, @name, and @age.
- A constructor that took name, age, and parent_permission as parameters. Both name and parent_permission were optional and had default values of "Unknown" and true, respectively.
- Getters were needed for accessing @id, @name, and @age.
- Setters were required for modifying @name and @age.
- A private method named of_age? was to be implemented, which would return true if @age was greater than or equal to 18; otherwise, it would return false.
- Additionally, a public method named can_use_services? was to be created. This method would return true if the person was of age or if they had obtained permission from their parents.

Furthermore, two additional classes were needed:

- A class called Student, which would inherit from the Person class. This class required the following features:
- A constructor that extended the parent's constructor by adding an instance variable @classroom along with a corresponding parameter.
- The implementation of a method named play_hooky, which would return the string "¯(ツ)/¯".
- A class called Teacher, which would also inherit from the Person class. This class needed the following features:
- A constructor that extended the parent's constructor by adding an instance variable @specialization along with a corresponding parameter.
- An override of the can_use_services? method to ensure that it always returned true.
- The project stipulated that each of these classes should be stored in separate files.

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require './person'
```
Comment (addod19) [inline student.rb:1]: ### Optional
- [x] It would be nice to use `require_relative` for files in our project directory

**Review by addod19 — CHANGES_REQUESTED** [review]
## Hi @sergio-usma, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- [ ] Person class has constructor set, public and private methods defined :rocket: 
- [ ] Correct Git-flow followed :100: 
- [ ] All classes have individual files :1st_place_medal: 
- [ ] The teacher and student classes both inherits from the person class :rocket: 
- [ ] The teacher class has a method that overrides the method in the Person class `can_use_sevices` :fireworks: 


## Required Changes ♻️
_Check the comments under the review._
- [ ] I'm leaving the comment here because in the file changes, i could not find the README file to leave this specific comment. In the README file, the link to the issues page is broken and returns `404` error page. Kindly fix it

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🚧 

Hi @sergio-usma 👋🏼 
I'm Andres.

You've done a great job implementing some of the requested changes for this milestone 🚀, but there are still some which haven't been addressed yet. You're getting closer, keep up the good work! 🤖 🎆 



 ## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 


 ## Required Changes ♻️
_Check the comments under the review._

- [x] According to the previous reviewer, please consider fixing your `Issues` link in your Readme file. Currently, it is not working, I suggest you implement it in this branch on a new commit. 🤓 🔧 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by iambenkis — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- All classes have individual files 🥇
- The teacher and student classes both inherits from the person class 🚀
- The teacher class has a method that overrides the method in the Person class can_use_sevices 🎆

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @iambenkis in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**